b2/account_info/in_memory.py

```diff
@@ -0,0 +1,91 @@
+######################################################################
+#
+# File: b2/account_info/in_memory.py
+#
+# Copyright 2016 Backblaze Inc. All Rights Reserved.
+#
+# License https://www.backblaze.com/using_b2_code.html
+#
+######################################################################
+
+from .exception import MissingAccountData
+from .upload_url_pool import UrlPoolAccountInfo
+
+from functools import wraps
+
+
+def _raise_missing_if_result_is_none(function):
+    @wraps(function)
+    def inner(*args, **kwargs):
+        result = function(*args, **kwargs)
+        if result is None:
+            assert function.__name__.startswith('get_')
```

Comment:
Putting the assertion outside the `if` would catch problems more reliably.

Better yet, what about putting it outside `inner`?